static/app/components/events/interfaces/spans/spanGroupBar.tsx

```diff
@@ -53,17 +53,24 @@ import {
   unwrapTreeDepth,
 } from './utils';
 
+export enum GroupType {
+  CHILDREN,
```

Comment:
@0Calories wondering if this is a better terminology for the pre-existing auto group feature I added. 

```suggestion
  DESCENDANTS,
```

Reply:
Yeah good point, I guess `CHILDREN` doesn't really imply that it applies to deeply nested spans 👍